core/src/main/scala/kafka/raft/KafkaMetadataLog.scala

```diff
@@ -581,10 +584,14 @@ object KafkaMetadataLog extends Logging {
     dataDir: File,
     time: Time,
     scheduler: Scheduler,
+    nodeId: Int,
+    maxBatchSizeBytes: Int,
```

Comment:
`maxBatchSizeBytes`, `maxFetchSizeBytes`, and `logDeleteDelayMs` can be moved to `MetadataLogConfig`.